assignments/objects.js

```diff
@@ -18,31 +18,67 @@ let example = {
 }
 
 // Write your intern objects here:
-
-
-// ==== Challenge 2: Reading Object Data ==== 
+const intern1 = {
+  id: 1,
+  name: 'Mitzi',
+  email: 'mmelloy0@psu.edu',
+  gender: 'F'
+};
+
+const intern2 = {
+  id: 2,
+  name: 'Kennan',
+  email: 'kdiben1@tinypic.com',
+  gender: 'M',
+  speak: 'Hello, my name is Kennan'
+};
+
+const intern3 = {
+  id: 3,
+  name: 'Keven',
+  email: 'kmummery2@wikimedia.org',
+  gender: 'M'
+};
+
+const intern4 = {
+  id: 4,
+  name: 'Gannie',
+  email: 'gmartinson3@illinois.edu',
+  gender: 'M'
+};
+
+const intern5 = {
+  id: 5,
+  name: 'Antonietta',
+  email: 'adaine5@samsung.com',
+  gender: 'F',
+  multiplyNums: 3*4
+};
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(intern1);
 // Kennan's ID
-
+console.log(intern2);
 // Keven's email
-
+console.log(intern3);
 // Gannie's name
-
+console.log(intern4);
 // Antonietta's Gender
```

Comment:
These functions would return all the information for each intern. To return a specific piece of info you'll use something like `console.log(intern1.name);`